createdisk.sh

```diff
@@ -283,8 +283,13 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl unmask chronyd
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl disable chronyd
 
 # Enable the io.podman.socket service
-${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl enable io.podman.socket
-
+if [[ ${OKD_VERSION} != "none" ]]
+then
+    # FCOS 33 drops the io. from podman.socket
```

Comment:
this is misleading, `io.podman.socket` is for varlink support, `podman.socket` is  for apiv2 support. `podman-2.0.5-5.module+el8.3.0+8221+97165c3f.x86_64` on my RHEL machine has both for example:
```
$ rpm -ql podman |grep socket
/usr/lib/systemd/system/io.podman.socket
/usr/lib/systemd/system/podman.socket
/usr/lib/systemd/user/io.podman.socket
/usr/lib/systemd/user/podman.socket
```

varlink support was dropped from recent podman versions, so `io.podman.socket` is gone. As I understand it, the plan was to make use of the newer API, so RHCOS should be fine too.
crc 1.19 (OpenShift 4.6.3) has `podman-1.9.3-3.rhaos4.6.el8.x86_64` which also has both socket files. I don't think this needs to be FCOS specific, the commit can just be "stop using varlink, it's gone from recent podman versions, use apiv2 instead"

Reply:
Agreed.  That is clearer.  I'll remove the OKD switch on that.

Reply:
Done.